### PR TITLE
Improve error message on components with invalid prop names

### PR DESCRIPTION
**Pull request by chriddyp**
https://github.com/plotly/dash/pull/1045 but rebased off of `dev`

### CODE REVIEW

**Comment by chriddyp** [comment]
Slight sizing issue in Percy which I'm assuming is irrelevant 
image

**Comment by chriddyp** [comment]
Tests are passing now 🎉

File: CHANGELOG.md [hunk 1]
```diff
@@ -9,6 +9,7 @@ This project adheres to [Semantic Versioning](http://semver.org/).
 
 ### Fixed
 - [#1037](https://github.com/plotly/dash/pull/1037) Fix no_update test to allow copies, such as those stored and retrieved from a cache.
+- [#1074](https://github.com/plotly/dash/pull/1045) Error messages when providing an incorrect property to a component have been improved: they now specify the component type, library, version, and ID (if available).
```
Comment (alexcjohnson) [inline CHANGELOG.md]: Not a big deal, but I wouldn't call this "Fixed", I'd call it "Changed." Fixed should refer to bugs only.

Comment (chriddyp) [reply CHANGELOG.md]: Fixed in 4abe0b8

**Comment by alexcjohnson** [comment]
> Slight sizing issue in Percy which I'm assuming is irrelevant

yeah that one has been doing that forever. would be great to figure out why and 🔒 it, but somehow the cause wasn't obvious to me last time I looked.

**Pull request merged**